### PR TITLE
New Flatpak for de.billardgl.Billardgl

### PR DESCRIPTION
**Pull request by ramcq**

### CODE REVIEW

**Comment by ramcq** [comment]
bot, build de.billardgl.Billardgl

**Comment by nedrichards** [comment]
I'm aware that this is pretty dead upstream but it'd be good to see an issue or email with the usual information about applied patches, offered desktop files and appdata and an offer to collaborate distributing on flathub.

File: de.billardgl.Billardgl.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <screenshot><image type="source">https://d3lapyynmdp1i9.cloudfront.net/screenshots/de.billardgl.Billardgl/C/de.billardgl.billardgl-screenshot3.jpg</image></screenshot>
+    </screenshots>
+
+    <content_rating type="oars-1.1" />
+
+    <metadata>
+        <value key="GnomeSoftware::popular-background">https://d3lapyynmdp1i9.cloudfront.net/thumbnails/de.billardgl.Billardgl/de.billardgl.billardgl-thumb.jpg</value>
+    </metadata>
+
+    <url type="homepage">http://www.billardgl.de/index-en.html</url>
```
[the remark this answers is not in the document]
Comment (nedrichards) [reply de.billardgl.Billardgl.appdata.xml]: That's not the correct link at all, it's an advert hosting page, the entire homepage thing should be removed.

File: de.billardgl.Billardgl.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{
+    "app-id": "de.billardgl.Billardgl",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+    "command": "billard-gl",
+    "build-options": {
```
Comment (nedrichards) [inline de.billardgl.Billardgl.json]: these are the default in 18.08 and should be removed here.

**Comment by ramcq** [comment]
> I'm aware that this is pretty dead upstream but it'd be good to see an issue or email with the usual information about applied patches, offered desktop files and appdata and an offer to collaborate distributing on flathub.

Done: https://sourceforge.net/p/billardgl/bugs/6/

**Comment by nedrichards** [comment]
bot, build de.billardgl.Billardgl

**Comment by ramcq** [comment]
bot, build de.billardgl.Billardgl

**Comment by ramcq** [comment]
Apparently you always need a homepage. :)

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/de.billardgl.Billardgl

**Pull request closed without merging**